compiler/one-cmds/one-quantize

```diff
@@ -512,6 +525,27 @@ def _copy_qparam(args):
         _utils._run(circle_quantizer_cmd, err_prefix="circle_quantizer", logfile=f)
 
 
+def _fake_quantize(args):
+    # get file path to log
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    logfile_path = os.path.realpath(args.output_path) + '.log'
+
+    with open(logfile_path, 'wb') as f:
+        # get driver path
+        circle_quantizer_path = os.path.join(dir_path, 'circle-quantizer')
+        q_model = ""
+        if _utils._is_valid_attr(args, 'input_path'):
+            q_model = getattr(args, 'input_path')
+        if _utils._is_valid_attr(args, 'output_path'):
+            fq_model = getattr(args, 'output_path')
```

Comment:
I think we don't have to call `if _utils._is_valid_attr():` for input_path nor output_path as these are checked at `_verify_arg()`

there are more codes that checks again in this file and would be, in another PR, OK to remove them.